.github/workflows/conda-build-test.yml

```
@@ -75,25 +58,16 @@ jobs:
           conda info
           conda list -n triton
 
-      - name: Build packages
-        if: ${{ steps.packages-cache.outputs.status == 'miss' }}
-        run: |
-          conda run --no-capture-output -n triton ./scripts/compile-triton.sh --llvm
-
-      - name: Save packages cache
-        if: ${{ steps.packages-cache.outputs.status == 'miss' }}
-        uses: ./.github/actions/save
-        with:
-          path: ${{ steps.packages-cache.outputs.path }}
-          dest: ${{ steps.packages-cache.outputs.dest }}
-
       - name: Build Triton
         run: |
-          conda run -n triton scripts/compile-triton.sh --triton
+          export DEBUG=1
+          cd python
+          conda run --no-capture-output -n triton pip install -e .
+          conda run --no-capture-output -n triton pip install -vvv -e '.[tests]'
```

As for as I understand, `pip install -e .` is not necessary if you do `pip install .[tests]`
```suggestion
          conda run --no-capture-output -n triton pip install -vvv -e '.[tests]'
```

---

Fixed